paper-drawer-panel.html

```diff
@@ -126,6 +134,8 @@
     }
 
     iron-selector > #main {
+      @apply(--paper-drawer-panel-main-container);
```

Comment:
I think we tend to move the user hook `@applys` to the last line in the block, so that they don't get overridden by the other style.